make/e2e.sh

```diff
@@ -170,22 +176,28 @@ if [[ "${ginkgo_args[*]}" =~ ginkgo.focus ]]; then
   ginkgo_args+=(--ginkgo.v --test.v)
 fi
 
+ginkgo_color=
+
+if ! should_color; then
+	ginkgo_color="--no-color"
+fi
+
 mkdir -p "$artifacts"
 
 export CGO_ENABLED=0
 
 trace ginkgo \
-  -tags=e2e_test \
-  -procs="$nodes" \
-  -output-dir="$artifacts" \
-  -junit-report="junit__01.xml" \
-  -flake-attempts="$flake_attempts" \
-  -timeout="24h" \
+  --tags=e2e_test \
+  --procs="$nodes" \
+  --output-dir="$artifacts" \
+  --junit-report="junit__01.xml" \
+  --flake-attempts="$flake_attempts" \
+  --timeout="1h" \
+  $ginkgo_color \
   -v \
-  -randomize-all \
-  -progress \
-  -trace \
-  -slow-spec-threshold="${GINKGO_SLOW_SPEC_THRESHOLD:-300s}" \
+  --randomize-all \
+  --trace \
+  --poll-progress-after=60s \
```

Comment:
Summary of changes:

1. Use `--` for long names, which seems to be what's documented. Probably not important, but I'd rather follow the style used in `ginkgo --help`.

2. Lowers overall timeout. 24h is far longer than we'd expect IMO.

3. Remove deprecated options "progress" and "slow-spec-threshold".

Reply:
Thank you for the changes!